src/Powercord/index.js

```diff
@@ -49,6 +50,10 @@ module.exports = class Powercord extends EventEmitter {
         SettingsManager.upload();
       }
     });
+
+    if (this.settings.get('hideToken')) {
```

Comment:
Don't forget default value
```suggestion
    if (this.settings.get('hideToken', true)) {
```